protocol-specs/generators/pos_random_seed_generation/index.js

```diff
@@ -131,7 +131,7 @@ const isValidSeedReveal = (seedReveal, previousSeedReveal) =>
 const selectSeedReveal = ({ fromHeight, toHeight, blocksMap, blocksPerRound }) => {
 	const selected = [];
 
-	for (let i = fromHeight; i >= toHeight; i -= 1) {
+	for (let i = fromHeight - 1; i >= toHeight; i -= 1) {
```

Comment:
Could you explain the logic here? I am not sure I understand.

Reply:
The change introduced in this PR make `getRandomSeed()` to include validator seed reveals that are strictly lower than `height + numberOfSeeds`. That's why it is needed to exclude the highest block from the parameters provided to `selectSeedReveal()` function, so the generator will exclude the highest block from the provided range, as per the LIP change.

Reply:
ah ok, I just saw the `i-=1`, makes perfect sense.